packages/core/src/components/TextField/TextField.tsx

```diff
@@ -146,6 +147,7 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
       iconsNames = EMPTY_OBJECT,
       type = TextFieldTextType.TEXT,
       maxLength = null,
+      allowExceedingLimit = false,
```

Comment:
Since it's tied to maxLength I wonder if it should be called allowExceedingMaxLength, though it's longer. WDYT?